common/thorhelper/thorcommon.hpp

```diff
@@ -89,7 +89,9 @@ enum RowReaderWriterFlags
     rw_compressblkcrc = 0x10, // block compression, this sets/checks crc's at block level
     rw_fastlz         = 0x20, // if rw_compress
     rw_autoflush      = 0x40,
-    rw_buffered       = 0x80
+    rw_buffered       = 0x80,
+    rw_lz4            = 0x100, // if rw_compress
+    rw_den            = 0x200  // if rw_compress
```

Comment:
as per above 2 comments, would be good to move these to a place where we can mask them.
Can safely move the 2 pre-existing flags (rw_compress and rw_compressblkcrc) without worrying about backward compatibility, as not used outside of the Thor instance build that uses them.